pkg/storage/etcd/etcd_helper.go

```diff
@@ -93,10 +137,10 @@ func NewEtcdStorage(client etcd.Client, codec runtime.Codec, prefix string, quor
 
 // etcdHelper is the reference implementation of storage.Interface.
 type etcdHelper struct {
-	etcdclient etcd.Client
-	client     etcd.KeysAPI
-	codec      runtime.Codec
-	copier     runtime.ObjectCopier
+	mapi   etcd.MembersAPI
```

Comment:
etcdMembers or etcdMembersClient or membersClient. Same for 'kapi'.

(IMO: short abbreviations help readability in short function local variables and hurt it in structs as member variables)